test/integration/simple_instance/inspec.yml

```diff
@@ -0,0 +1,27 @@
+name: simple_instance
+attributes:
```

Comment:
Kitchen-terraform will pass through Terraform outputs as InSpec attributes, but InSpec still needs them declared. Earlier versions of InSpec declared variables in *each* control file with the following syntax:

```ruby
project_id = attribute('project_id', required: true, type: :string)
group_email = attribute('group_email', default: nil)
group_role = attribute('group_role', default: nil)
service_account_email = attribute('service_account_email', required: true, type: :string)
extra_service_account_email = attribute('extra_service_account_email', required: true, type: :string)
```
(Example taken from https://github.com/terraform-google-modules/terraform-google-project-factory/pull/34/files#diff-ace4cdc8cd5b26e29c2cb7c34d81103c)

InSpec added global attributes in https://github.com/inspec/inspec/pull/3318 which allows us to provide a single definition for attributes in this YAML file, which means that controls can simply use attributes by calling `attribute('name')` instead of using the old style that both declared attributes and then looked them up.

So in this case kitchen-terraform is passing everything through, but InSpec still needs to know what attributes are defined and can be looked up.